Gemfile

```diff
@@ -10,7 +10,7 @@ gem 'jbuilder', '~> 2.0'
 gem 'jquery-rails', '4.3.1'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '4.2.0'
+gem 'rails', '4.2.1'
```

Comment:
Do you think we can use the latest rails v4.2.x to benefit from the bug fixes and security patches?

Reply:
I have ran tests against all versions up to 4.2.10 (latest I believe of 4.2 branch) and all is fine.  Ive even got 5 working but thats probably going too far